site/app/models/gradeable/GradedGradeable.php

```diff
@@ -133,11 +133,12 @@ public function isTaGradingComplete() {
     /**
      * Gets whether a peer grader has graded all of the peer components for this submitter/gradeable
      * Later this will take in a userId and determine if that user graded all components
+     * @param User|null $grader Peer grader to check if all peer components associated with this grader has been graded.
      * @return bool
      */
-    public function isPeerGradingComplete() {
+    public function isPeerGradingComplete(User $grader = null) {
```

Comment:
Let add return type annotation while editing this function, and apply throughout